src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.ps1

```diff
@@ -0,0 +1,127 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+CortexCRUD
+#>
+function Test-CortexCRUD
+{
+ # Setup
+    $rgName = Get-ResourceName
+    $rglocation = Get-Location Microsoft.Network virtualWans eastus2euap
+
+	$virtualWanName = Get-ResourceName
+	$virtualHubName = Get-ResourceName
+	$vpnSiteName = Get-ResourceName
+	$vpnGatewayName = Get-ResourceName
+	$remoteVirtualNetworkName = Get-ResourceName
+	$vpnConnectionName = Get-ResourceName
+	$hubVnetConnectionName = Get-ResourceName
+    
+	try
+	{
+		# Create the resource group
+        $resourceGroup = New-AzureRmResourceGroup -Name $rgName -Location $rglocation
+
+		# Create the Virtual Wan
+		$createdVirtualWan = New-AzureRmVirtualWan -ResourceGroupName $rgName -Name $virtualWanName -Location $rglocation -AllowVnetToVnetTraffic $true -AllowBranchToBranchTraffic $true
+		$createdVirtualWan = Update-AzureRmVirtualWan -ResourceGroupName $rgName -Name $virtualWanName -AllowVnetToVnetTraffic $false -AllowBranchToBranchTraffic $false
+		$virtualWan = Get-AzureRmVirtualWan -ResourceGroupName $rgName -Name $virtualWanName
+		Assert-AreEqual $rgName $virtualWan.ResourceGroupName
+		Assert-AreEqual $virtualWanName $virtualWan.Name
```

Comment:
None of the asserts in the test are testing the values of the properties that you are explicitly setting, so it is hard to tell whether the Update cmdlet is working.  Please add more asserts for all the cmdlets.

Reply:
Fixed